content/doc/book/installing/_kubernetes.adoc

```diff
@@ -552,3 +553,183 @@ This may also be found at:
+
+Configure Helm::
+Once Helm is installed and set up properly, add the Jenkins-Operator repo as follows:
+
+[source,bash]
+----
+$ helm repo add jenkins https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/chart 
+----
+
+Install Jenkins Operator::
+
+[source,bash]
+----
+$ helm install <name> jenkins/jenkins-operator -n jenkins
+----
+
+To add custom labels and annotations, you can use `values.yaml` file as explained in the link:/doc/book/installing/kubernetes/#install-jenkins[Install Jenkins] section above or pass them into helm install command, e.g.:
+
+[source,bash]
+----
+$ helm install <name> jenkins/jenkins-operator -n jenkins --set jenkins.labels.LabelKey=LabelValue,jenkins.annotations.AnnotationKey=AnnotationValue
+----
+
+===== Deploy Jenkins
+Once Jenkins Operator is up and running let’s deploy actual Jenkins instance. 
+Create a manifest e.g. jenkins_instance.yaml with following data and save it locally.
+
+[source,yaml]
+----
+apiVersion: jenkins.io/v1alpha2
+kind: Jenkins
+metadata:
+  name: example
+spec:
+  master:
+    containers:
+    - name: jenkins-master
+      image: jenkins/jenkins:lts
+      imagePullPolicy: Always
+      livenessProbe:
+        failureThreshold: 12
+        httpGet:
+          path: /login
+          port: http
+          scheme: HTTP
+        initialDelaySeconds: 80
+        periodSeconds: 10
+        successThreshold: 1
+        timeoutSeconds: 5
+      readinessProbe:
+        failureThreshold: 3
+        httpGet:
+          path: /login
+          port: http
+          scheme: HTTP
+        initialDelaySeconds: 30
+        periodSeconds: 10
+        successThreshold: 1
+        timeoutSeconds: 1
+      resources:
+        limits:
+          cpu: 1500m
+          memory: 3Gi
+        requests:
+          cpu: "1"
+          memory: 500Mi
+  seedJobs:
+  - id: jenkins-operator
+    targets: "cicd/jobs/*.jenkins"
+    description: "Jenkins Operator repository"
+    repositoryBranch: master
+    repositoryUrl: https://github.com/jenkinsci/kubernetes-operator.git
+----
+
+===== Deploy Jenkins to Kubernetes:
+
+[source,bash]
+----
+$ kubectl create -f jenkins_instance.yaml -n jenkins
+----
+
+Watch the Jenkins instance being created:
+
+[source,bash]
+----
+$ kubectl get pods -n jenkins
+----
+
+Get Jenkins credentials::
+
+[source,bash]
+----
+$ kubectl get secret jenkins-operator-credentials-example -o 'jsonpath={.data.user}' | base64 -d
+$ kubectl get secret jenkins-operator-credentials-example -o 'jsonpath={.data.password}' | base64 -d
+----
+
+Connect to Jenkins (minikube)::
+
+[source,bash]
+----
+$ minikube service jenkins-operator-example --url
+----
+
+Connect to Jenkins (actual Kubernetes cluster)::
+
+[source,bash]
+----
+$ kubectl describe svc jenkins-operator-example
+$ kubectl port-forward jenkins-operator-example 8080:8080
+----
+
+Then open browser with address http://localhost:8080 to view your Jenkins Dashboard.
```

Comment:
`Then open browser with address http://localhost:8080 to view your Jenkins Dashboard.` should read `Then open a browser with the address http://localhost:8080 to view your Jenkins Dashboard.`